deployments/examples/owncloud10_with_oc_web/config/ocis/identifier-registration.dist.yaml

```
@@ -34,7 +34,9 @@ clients:
   - id: xdXOt13JKxym1B1QcEncf2XDkLAexMBFwiT9j6EfhhHFJhs2KM9jbjTmf8JBXE69
     secret: UBntmLjC2yYCeHwsyj73Uwo9TAaecAetRwMw0xYcvNL9yRdLSUi0hUAHfvCHFeFh
```

```suggestion
    secret: UBntmLjC2yYCeHwsyj73Uwo9TAaecAetRwMw0xYcvNL9yRdLSUi0hUAHfvCHFeFh
    name: ownCloud desktop app
```

No idea why the name got lost here.